src/select-field.jsx

```diff
@@ -82,7 +82,7 @@ let SelectField = React.createClass({
   },
 
   onChange(e, index, payload) {
-    if (payload) {
+    if (payload && this.props.valueMember) {
       e.target.value = payload[this.props.valueMember] || payload;
     }
```

Comment:
Yeh, it does seem weird that `SelectField` is having to handle `onChange`. It seems like it should be handled in one place?